### PR TITLE
unauthorized action changed 401 status code to 403

### PR DESCRIPTION
**Pull request by ilkerCelimli**
#59 unauthorized action changed 401 status code to 403

### CODE REVIEW

**Review by skayikci — APPROVED** [review]

File: src/main/java/com/ays/common/util/exception/model/AysError.java [hunk 1]
```diff
@@ -113,7 +113,12 @@ public enum Header {
         /**
          * AUTH_ERROR header.
          */
-        AUTH_ERROR("AUTH ERROR");
+        AUTH_ERROR("AUTH ERROR"),
+
+        /**
+         * FORBIDDEN_ERROR header.
+         */
+        FORBIDDEN_ERROR("FORBIDDEN ERROR");
```
Comment (agitrubard) [inline src/main/java/com/ays/common/util/exception/model/AysError.java:121]: @ilkerCelimli, we can remove the header here and use the `AUTH_ERROR` header in the same way we use it in the application, even in the test side.

File: src/main/java/com/ays/auth/security/CustomAuthenticationEntryPoint.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                .httpStatus(HttpStatus.UNAUTHORIZED)
+                .isSuccess(false)
+                .build();
+        final String responseBody = OBJECT_MAPPER
+                .writer(DateFormat.getDateInstance())
+                .writeValueAsString(error);
+        httpServletResponse.getOutputStream().write(responseBody.getBytes());
+    }
+
+}
```
Comment (agitrubard) [inline src/main/java/com/ays/auth/security/CustomAuthenticationEntryPoint.java:59]: @ilkerCelimli @Rapter1990 I reviewed the class you provided. I made some code modifications and you can review them as well.

File: docker-compose.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ services:
     build: .
     ports:
       - "8080:8080"
+
```
Comment (agitrubard) [inline docker-compose.yml:8]: @ilkerCelimli Can we revert the change here? There shouldn't have been a need for such a modification specific to the development.

File: .gitignore [hunk 4]
```diff
@@ -60,5 +60,5 @@ build/
 
 ## Local env file having environment variable definitions for docker-compose.yml and other files
 ## We should not commit this to prevent leaking confidential information
-.env*
+.env
```
Comment (agitrubard) [inline .gitignore:63]: @ilkerCelimli Can we revert the change here? There shouldn't have been a need for such a modification specific to the development. Especially the changes made in the `application.yml` were preventing the application from being launched, so I had to remove them.

**Review by agitrubard — CHANGES_REQUESTED** [review]

**Review by agitrubard — APPROVED** [review]

**Pull request merged**